pkg/client/listers_generated/servicecatalog/v1beta1/mocks_generate.go

```diff
@@ -0,0 +1,10 @@
+package v1beta1
```

Comment:
Is this file manually created? I am unclear how the mock lister is generated. If this file is manually created, then it is in conflict with the warning [here](https://github.com/kubernetes-incubator/service-catalog/blob/master/pkg/client/WARNING.md).